packages/core/src/web/app/views/beambox/Right-Panels/ActionsPanel.tsx

```diff
@@ -175,26 +213,39 @@ const ActionsPanel = ({ elem }: Props): React.JSX.Element => {
       { isFullLine: true, ...opts },
     );
 
-  const renderTabButton = (opts: ButtonOpts = {}): React.JSX.Element =>
-    renderButtons(
+  const renderTabButton = (
+    { convertToPath, ...options }: TabButtonOptions = {
+      convertToPath: convertSvgToPath,
+    },
+  ): React.JSX.Element => {
+    const isDisabled = match(elem.getAttribute('fill'))
+      // 'none' for SVG elements
+      // 'nullish' for use elements
+      .with(P.union('none', P.nullish), () => false)
+      .otherwise(() => true);
+
+    return renderButtons(
       'tab',
       tab,
-      () => {
-        const bbox = (elem as SVGSVGElement).getBBox();
+      async () => {
+        let bbox = (elem as SVGSVGElement).getBBox();
+        let command: IBatchCommand | undefined;
 
         // Convert to path if it's not a path
         if (!(elem instanceof SVGPathElement)) {
-          const { path } = svgCanvas.convertToPath(elem as SVGElement);
+          const { bbox: newBbox, command: subCommand } = await convertToPath();
 
-          svgCanvas.selectOnly([path]);
+          bbox = newBbox;
+          command = subCommand;
```

Comment:
if someone open the tab panel but end up canceling the operation, the object would still be convert to path, is that desired?